luxonis_train/nodes/backbones/ddrnet/ddrnet.py

```diff
@@ -0,0 +1,298 @@
+"""DDRNet backbone.
```

Comment:
This can be moved to the class docstring. Also add the original source and paper links under `@see` tags .
(like `@see: U{Paper <https://...>}`)